src/Text/Pandoc/Writers/LaTeX.hs

```diff
@@ -65,10 +65,37 @@ import Text.Pandoc.Writers.LaTeX.Util (stringToLaTeX, StringContext(..),
 import Text.Pandoc.Writers.Shared
 import qualified Text.Pandoc.Writers.AnnotatedTable as Ann
 
```

Comment:
It would be useful to add a comment here like
```
-- Work around problems with notes inside emphasis (see #8982)
```

Reply:
Added!